src/Libraries/PythonNodeModelsWpf/PythonNode.cs

```diff
@@ -62,6 +63,10 @@ public void CustomizeView(PythonNode nodeModel, NodeView nodeView)
             nodeView.MouseDown += View_MouseDown;
             nodeModel.DeletionStarted += NodeModel_DeletionStarted;
             nodeModel.Disposed += NodeModel_Disposed;
+
+            nodeView.PresentationGrid.Visibility = Visibility.Visible;
+            nodeView.PresentationGrid.DataContext = this.pythonNodeModel;
```

Comment:
I think usually the datacontext is the viewModel - I'm not sure it matters - but I think thats the pattern in most of the repo - does that sufficiently complicate things?

Reply:
I see, this was only so i could bind to the Engine property on the node model. I changed it so im initializing the `EngineLabel` with the Node model so i can reference that directly from the xaml.

- https://github.com/DynamoDS/Dynamo/pull/10607/files#diff-dc0bc4a072798ad7f020dc869621e9c2R68 

- https://github.com/DynamoDS/Dynamo/pull/10607/files#diff-ccc4c3c093f111d95e8c19e41880fe75R28)